src/main/java/guessingnumber/AnswerGenerator.java

```diff
@@ -0,0 +1,10 @@
+package guessingnumber;
+
+import java.util.Random;
+
+public class AnswerGenerator {
+    private static final Random RANDOM_GENERATOR = new Random();
+    public int getAnswer(){
+        return RANDOM_GENERATOR.nextInt(100) + 1;
+    }
+}
```

Comment:
```suggestion
public class AnswerGenerator {

    private static final Random RANDOM_GENERATOR = new Random();
    
    public int getAnswer(){
        return RANDOM_GENERATOR.nextInt(100) + 1; // 100, 1이 의미하는 것이 무엇일까요??
    }
}
```
해당 리뷰는 다른 모든 클래스에서도 동일하게 적용됩니다.

적절한 공백은 다른 개발자로 하여금 코드를 이해하기 쉽게 만들어줘요~
상수, 인스턴스 변수, 생성자, 메서드 사이에 적절한 공백을 추가하여 가독성을 높이는 것이 좋겠네요! 👍

---

100, 1이 의미하는 바는 무엇일까요??
100과 1이 의미하는 바를 명확히 하기 위해 상수로 정의해보는 것은 어떨까요? 😀